pillar/heroku/discussions.sls

```diff
@@ -108,6 +111,8 @@ heroku:
   app_name: {{ env_data.app_name }}
   api_key: __vault__::secret-operations/global/heroku/api_key>data>value
   config_vars:
+    AKISMET_API_KEY: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/akismet>data>api_key
```

Comment:
If CI api_key is gonna be different than RC, then this should be changed from `env_data.vault_env_path` to `env_data.env_name`

Reply:
done